### PR TITLE
Add sample for sentiment analysis

### PR DESCRIPTION
**Pull request by LakshanWeerasinghe**
## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,6 +16,7 @@ Each subdirectory contains relevant code and instructions for the corresponding
 - [Clean and Standardize Data](clean-and-standardize-data): Sample code for cleaning and standardizing data.
 - [Data Mapper](data-mapper): Sample code for data mapping between different structures.
 - [Enrich Data](enrich-data): Sample code to enrich data.
+- [Sentiment Analysis](sentiment-analysis): Sample code for analyse feedbacks.
```
Comment (pcnfernando) [inline README.md:18]: Shall we remove the earlier one?

File: sentiment-analysis/main.bal [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+type SentimentResponse record {
+    DocumentSentiment documentSentiment;
+};
+
+type DocumentSentiment record {|
+    float magnitude;
+    float score;
+|};
+
+final http:Client sentimentClient = check new ("https://language.googleapis.com");
```
Comment (pcnfernando) [inline sentiment-analysis/main.bal]: This sample is then again another HTTP EP integration; my suggestion was to use openAPI client to do the sentiment analysis. Please refer https://ballerina.io/usecases/ai/

Comment (LakshanWeerasinghe) [reply sentiment-analysis/main.bal]: updated.

File: README.md (hunk 1 above)
Comment (LakshanWeerasinghe) [reply README.md:18]: removed from the README but didn't remove the sample.

**Comment by LakshanWeerasinghe** [comment]
Closing this PR since we already have [sample](https://github.com/ballerina-guides/etl-samples/blob/main/refine-data-using-ai/main.bal) with openapi connector.

**Pull request closed without merging**